src/pystatis/db.py

```diff
@@ -28,6 +29,35 @@ def identify_db(name: str) -> list[str]:
     return db_match
 
 
+def identify_db(table_name: str) -> str:
```

Comment:
this method should be in db.py not in table.py in my opinion. Didn't we had this function before?